konfera/models/discountcodes.py

```diff
@@ -1,11 +1,19 @@
+from django.core.exceptions import ValidationError
 from django.db import models
-from .tickettype import TicketType
+from konfera.models import TicketType
+
+
+def validate_min_max(value):
+    if value < 0 or value > 100:
+        raise ValidationError('{} not valid Discount value'.format(value),
+                              params={'value': value},
+                              )
 
 
 class DiscountCodes(models.Model):
     title = models.CharField(max_length=128)
     hash = models.CharField(max_length=64)
-    discount = models.IntegerField(max(100), min(0))
+    discount = models.IntegerField(validators=[validate_min_max])   # TODO validation test needed
```

Comment:
I done think we need a custom validator method for this, how about we use Django build in validators:

```python
from django.core.validators import MaxValueValidator, MinValueValidator

    discount = models.IntegerField(
        default=0,
        validators=[
            MaxValueValidator(100),
            MinValueValidator(0)
        ]
     )
```